src/main/java/com/sparta/gamjaquick/menu/dto/response/MenuDeleteReponseDto.java

```diff
@@ -0,0 +1,23 @@
+package com.sparta.gamjaquick.menu.dto.response;
+
+import com.sparta.gamjaquick.menu.dto.request.MenuRequestDto;
+import com.sparta.gamjaquick.menu.entity.Menu;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@NoArgsConstructor
+@Getter
+public class MenuDeleteReponseDto {
+    private UUID menuId;
+    private LocalDateTime deletedAt;
+    private String deletedBy;
+
+    public MenuDeleteReponseDto(UUID id, LocalDateTime deletedAt, String deletedBy) {
+        this.menuId = id;
+        this.deletedAt = deletedAt;
+        this.deletedBy = deletedBy;
+    }
+}
```

Comment:
해당 부분은 lombok의 @AllArgsConstructor를 붙이는 것도 방법일 것 같습니다!
```suggestion
    @AllArgsConstructor
    @NoArgsConstructor
    @Getter
    public class MenuDeleteReponseDto {
```

Reply:
헐 바보 감자